sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_arg.asciidoc

```diff
@@ -386,6 +511,36 @@ constant property.
 
 |===
 
+The following template implements the overloaded operators for types
+`annotated_arg<T, ...>` and `annotated_arg<T2, ...>`. Let `operatorOP` denote
+the operator used. The overloaded operator `operatorOP` utilizes `operatorOP(T2)`
+of type `T` and is available only if `operatorOP(T2)` is valid for objects of type `T`.
+The value and result is the same as the result of `operator+(T2)` applied to the
+objects of type `T` and `T2`.
+
+[source,c++]
+----
+template <typename T2, typename PropertyList2,
+          typename R = decltype(std::declval<T>() OP std::declval<T2>())>
+R operatorOP(const annotated_arg<T2, PropertyList2> &other) const;
+----
+
+The following templates implement the overloaded operators for types `annotated_arg<T, ...>`
```

Comment:
For the same reason as above, this is a little confusing to read.  It sounds like you're saying that this is the code itself that will be in the header file as opposed to a collapsed representation of it.  Maybe go with:

```suggestion
Free functions of the following form are defined for the `overloaded operators for types `annotated_arg<T, ...>`
```

And, as above, I think you should work in the list of functions somewhere.